configure.py

```diff
@@ -1438,7 +1453,7 @@ def configure_zstd(build_dir, mode):
         rule configure
           command = {python} configure.py $configure_args
           generator = 1
-        build build.ninja: configure | configure.py seastar/configure.py
+        build build.ninja: configure | configure.py seastar/CMakeLists.txt
```

Comment:
I think the `seastar/*` dependency is unnecessary in this rule. We already unconditionally invoke `ninja .../libseastar.a` regardless of the target built. CMake will detect if any of its `CMakeLists.txt` files is dirty and will re-run itself.

Reply:
ok, will drop (it's very partial in any case)

Reply:
Yes, there is a CMakeLists.txt in every directory, not just the top-level one.